app/views/common/403.html.erb

```diff
@@ -2,7 +2,14 @@
   <h2><%= _("Permission denied") %></h2>
 
   <p><strong><%= _("You are not authorized to perform this action") %></strong></p>
-  <p><%= _("Please request the required privileges from a foreman administrator") %></p>
+  <p><%= _("Please request the one of required privileges from a foreman administrator listed below") %>
```

Comment:
a) capitalise "Foreman"
b) missing punctuation at the end, it probably should have a `:`
c) "Please request one of the required permissions"